extension.driver.php

```diff
@@ -703,7 +703,7 @@ public function appendFilter($context) {
 				);
 			}
 
-			if(!FieldManager::isFieldUsed(self::getFieldType('authentication'))) {
+			if(FieldManager::isFieldUsed(self::getFieldType('authentication'))) {
```

Comment:
This does not make it show "always", but only if a authentication field is used. It was a bug ?

Reply:
Good catch! Yes, I can only suspect that this was a bug, introduced by accident. The filter will call the `filter_UpdatePasswordLogin` function, which is completely useless without an authentication field. So the filter itself has been useless all the time, and I don't consider this a breaking change but a bugfix. (Please, confirm! I don't want to break stuff, of course.)

For the same reason showing the filter is only useful if an authentication field exists, so IMHO "always if the field exists" is enough.

Reply:
LGTM then!

Reply:
BTW, I found the misunderstanding. In https://github.com/symphonycms/members/issues/44#issuecomment-15017336 @jensscherbl said:

> Added this as a filter (Members: Login), which is only available if no activation is needed. Just like @brendo suggested.

So the idea was to only show the filter if no activation field exists. But actually he tested for the authentication field. I don't know why nobody ever noticed this.

(I don't think that we need to test for the activation field. It's up to the developer to use the the filter (and see if it works). The filter just attempts what the name says: Log the Member in.)

Reply:
Good.